README.md

```diff
@@ -4,25 +4,25 @@ We will be using git as version control this semester in order to keep track of
 ## Basic Commands
 git init - initialize a git repository. You must do this at the top level directory of every new project you create. You don't need to do this if you clone someone else's repository.
 
-git clone <url> - Clone a repository from github. 
+git clone `<url>` - Clone a repository from github. 
 
-git remote add origin <url> - Sets the url origin of your git repository for when you push to github.
+git remote add origin `<url>` - Sets the url origin of your git repository for when you push to github.
```

Comment:
Thank you for writing this out.

I think having the whole command in `backquotes` is visually helpful. For example `git clone <url>` or `git remote add origin <url>`